### PR TITLE
Feature/offpay 568

### PR DESCRIPTION
**Pull request by habeeb01**

### CODE REVIEW

File: test/uk/gov/hmrc/decisionservice/controllers/ControlAggregatedCsvSpec.scala [hunk 1]
```diff
@@ -20,7 +20,7 @@ import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
 
 class ControlAggregatedCsvSpec extends UnitSpec with WithFakeApplication with DecisionControllerClusterCsvSpec {
   val clusterName = "control"
-  val CONTROL_SCENARIOS = "/test-scenarios/single/control/scenarios.csv"
+  val CONTROL_SCENARIOS = "/test-scenarios/single-DELETE/control/scenarios.csv"
```
Comment (miloszm) [inline test/uk/gov/hmrc/decisionservice/controllers/ControlAggregatedCsvSpec.scala]: does this path exist? shouldn't it go away

File: test/uk/gov/hmrc/decisionservice/controllers/ControlCsvSpec.scala [hunk 2]
```diff
@@ -21,39 +21,41 @@ import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
...
-    s"return 200 and correct response control scenario 3 for version ${Versions.VERSION101_BETA}" in {
-      createRequestSendVerifyDecision(CONTROL_SCENARIO_3, Versions.VERSION101_BETA)
-    }
-    s"return 200 and correct response control scenario 0 for version ${Versions.VERSION100_FINAL}" in {
-      createRequestSendVerifyDecision(CONTROL_SCENARIO_0_VERSION2, Versions.VERSION100_FINAL)
-    }
-    s"return 200 and correct response control scenarios for version ${Versions.VERSION100_FINAL}" in {
-      createMultipleRequestsSendVerifyDecision(CONTROL_SCENARIOS_VERSION2, Versions.VERSION100_FINAL)
+
+    //v1.1.0-final
```
Comment (miloszm) [inline test/uk/gov/hmrc/decisionservice/controllers/ControlCsvSpec.scala]: I think comments like that could/should go away - text used in tests bodies is or ought to be sufficient

File: test/uk/gov/hmrc/decisionservice/controllers/MatrixCsvSpec.scala [hunk 3]
```diff
@@ -20,43 +20,33 @@ import uk.gov.hmrc.decisionservice.Versions
...
-    s"return 200 and correct response with the inside IR35 decision for version ${Versions.VERSION101_BETA}" in {
-      createRequestSendVerifyDecision(TEST_CASE_INSIDE_IR35, Versions.VERSION101_BETA)
-    }
-    s"return 200 and correct response with the inside IR35 decision (matches Financial Risk Blank) for version ${Versions.VERSION101_BETA}" in {
-      createRequestSendVerifyDecision(TEST_CASE_INSIDE_IR35_MATCH_FINANCIAL_RISK_BLANK, Versions.VERSION101_BETA)
-    }
-    s"return 200 and correct response with the unknown decision for version ${Versions.VERSION101_BETA}" in {
-      createRequestSendVerifyDecision(TEST_CASE_UNKNOWN, Versions.VERSION101_BETA)
+
+    //v1.1.0-final
```
Comment (miloszm) [inline test/uk/gov/hmrc/decisionservice/controllers/MatrixCsvSpec.scala]: I am not sure we should rely on comments like this

File: test/uk/gov/hmrc/decisionservice/controllers/PersonalServiceAggregatedCsvSpec.scala [hunk 4]
```diff
@@ -21,11 +21,11 @@ import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
 
 class PersonalServiceAggregatedCsvSpec extends UnitSpec with WithFakeApplication with DecisionControllerClusterCsvSpec {
   val clusterName = "personalService"
-  val PERSONAL_SERVICE_SCENARIOS = "/test-scenarios/single/personal-service/scenarios.csv"
+  val PERSONAL_SERVICE_SCENARIOS = "/test-scenarios/single-DELETE/personal-service/scenarios.csv"
 
   "POST /decide" should {
     "return 200 and correct response with the expected decisions for personal service scenarios" in {
-      createMultipleRequestsSendVerifyDecision(PERSONAL_SERVICE_SCENARIOS, Versions.VERSION101_BETA)
+      //FIXME add tests for 1.1.0, 1.1.1 and 1.2.0
```
Comment (miloszm) [inline test/uk/gov/hmrc/decisionservice/controllers/PersonalServiceAggregatedCsvSpec.scala]: maybe we should add JIRA backlog item here

File: test/uk/gov/hmrc/decisionservice/controllers/ControlAggregatedCsvSpec.scala (hunk 1 above)
Comment (habeeb01) [inline test/uk/gov/hmrc/decisionservice/controllers/ControlAggregatedCsvSpec.scala]: I forgot to delete ControlAggregatedCsvSpec. ControlCsvSpec now includes all the tests.

File: test/uk/gov/hmrc/decisionservice/controllers/ControlCsvSpec.scala (hunk 2 above)
Comment (habeeb01) [inline test/uk/gov/hmrc/decisionservice/controllers/ControlCsvSpec.scala]: removed

File: test/uk/gov/hmrc/decisionservice/controllers/MatrixCsvSpec.scala (hunk 3 above)
Comment (habeeb01) [inline test/uk/gov/hmrc/decisionservice/controllers/MatrixCsvSpec.scala]: comment removed

File: test/uk/gov/hmrc/decisionservice/controllers/PersonalServiceAggregatedCsvSpec.scala (hunk 4 above)
Comment (habeeb01) [inline test/uk/gov/hmrc/decisionservice/controllers/PersonalServiceAggregatedCsvSpec.scala]: PersonalServiceAggregatedCsvSpec has also been removed. PersonalServiceCsvSpec now has all the tests.

**Pull request merged**